TitaniumTask/app/src/main/java/com/sa/testtask/screens/MainActivity.java

```diff
@@ -97,13 +93,6 @@ private void Subscribe() {
         subscription.add(onSaveClick());
         subscription.add(onRandomClick());
         subscription.add(onSelectClick());
-
-    }
-
-    @Override
```

Comment:
why is it not necessary? If screen orientation will change Activity will be destroyed and will show image that came when Activity created the first time